lib/pbench/server/database/models/metadata.py

```diff
@@ -0,0 +1,88 @@
+import enum
+import datetime
+from pbench.server.database.database import Database
+from sqlalchemy import Column, Integer, DateTime, ForeignKey, Text
+from sqlalchemy.orm import validates
+
+
+class MetadataKeys(enum.Enum):
+    FAVORITE = 1
+    SAVED = 2
```

Comment:
Ah; I see why you chose to uppercase the metadata key. I'm not so happy about persisting them as integers; it'd be cleaner to use an `Enum(MetadataKeys)` column type, if for no other reason than it's a lot easier to read the SQL output from a database for debugging.

Reply:
I got rid of enums to keep the metadata key flexible and not restricted to certain values